CMakeLists.txt

```diff
@@ -128,7 +123,9 @@ endif()
 
 if (AMR_WIND_ENABLE_UNIT_TESTS OR AMR_WIND_ENABLE_TESTS)
   add_executable(${amr_wind_unit_test_exe_name})
+  disable_clang_tidy()
   add_subdirectory("submods/googletest")
+  enable_clang_tidy()
```

Comment:
Instead of the toggle, is it possible to use `set_target_properties` to only add the `CXX_CLANG_TIDY` property to AMR-Wind targets?

```cmake
set_target_properties(${amr_wind_lib_name} PROPERTIES CXX_CLANG_TIDY ${CLANG_TIDY_EXE})
```

Reply:
Oh that's a good idea. I was trying to think of how to do this originally, but I kept thinking of it as having to turn off `${CLANG_TIDY_EXE}` for all other targets.